test/skate_web/controllers/auth_controller_test.exs

```diff
@@ -103,6 +106,49 @@ defmodule SkateWeb.AuthControllerTest do
     end
   end
 
+  describe "GET /auth/keycloak/logout" do
+    test "redirects to `sign_out_url`", %{conn: conn} do
+      redirect_url = "redirect.url.localhost"
+
+      reassign_env(:skate, :logout_url_fn, fn _ ->
+        {:ok, redirect_url}
+      end)
+
+      conn =
+        conn
+        |> init_test_session(%{})
+        |> assign(:ueberauth_auth, Skate.Factory.build(:ueberauth_auth))
+        |> get(~p"/auth/keycloak/callback")
+
+      assert Guardian.Plug.authenticated?(conn)
+
+      conn = get(conn, ~p"/auth/keycloak/logout")
+
+      refute Guardian.Plug.authenticated?(conn)
```

Comment:
**nitpick (non-blocking):** I guess we don't _strictly_ need these Guardian-related assertions in this test since other tests cover the Guardian functionality, but I suppose they're nice guardrails to make sure that the sign in / out flow works, so up to you on keeping them or not.